bin/sozo/src/commands/build.rs

```diff
@@ -9,6 +9,11 @@ use scarb::ops::{CompileOpts, FeaturesOpts, FeaturesSelector};
 use sozo_ops::statistics::{get_contract_statistics_for_dir, ContractStatistics};
 use tracing::trace;
 
+const BYTECODE_SIZE_LABEL: &str = "Bytecode size [in felts]\n(Sierra, Casm)";
```

Comment:
```suggestion
const BYTECODE_SIZE_LABEL: &str = "Bytecode size\n(Sierra, Casm)";
```
We can remove those then, right?

Reply:
i think we should keep those units, since its easier to read that compared to the explanation above.

Reply:
Oh I thought that the text will replace the units, all good then. 👍